sceptre/template_handlers/s3.py

```diff
@@ -22,11 +31,51 @@ def schema(self):
         }
 
     def handle(self):
-        self.logger.debug("Downloading file from S3: %s" % self.arguments["path"])
-
+        logger.debug(f'test200')
         segments = self.arguments["path"].split('/')
         bucket = segments[0]
         key = "/".join(segments[1:])
+        filename = segments[-1]
+        extension = filename.split('.')[1]
+        file_extension = f'.{extension}'
```

Comment:
You could probably use pathlib.Path and get a lot of this extension stuff for free.